pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -272,9 +287,14 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) updateStatus(client client.Clien
 	}
 
 	cephFilesystemSubVolumeGroup.Status.Phase = status
+	cephFilesystemSubVolumeGroup.Status.Info = map[string]string{"clusterID": buildClusterID(cephFilesystemSubVolumeGroup)}
 	if err := reporting.UpdateStatus(client, cephFilesystemSubVolumeGroup); err != nil {
 		logger.Errorf("failed to set ceph ceph filesystem subvolume group %q status to %q. %v", name, status, err)
 		return
 	}
 	logger.Debugf("ceph ceph filesystem subvolume group %q status updated to %q", name, status)
 }
+
+func buildClusterID(cephFilesystemSubVolumeGroup *cephv1.CephFilesystemSubVolumeGroup) string {
+	return fmt.Sprintf("%s-%s-file-%s", cephFilesystemSubVolumeGroup.Namespace, cephFilesystemSubVolumeGroup.Spec.FilesystemName, cephFilesystemSubVolumeGroup.Name)
```

Comment:
as discussed this should be hash or else we will hit clusterID length limit issue in cephcsi.

Reply:
Or right, the hash 🤦🏻 , I can easily add this!

Reply:
What's the char length limit already?

Reply:
clusterID length limit is 64 .  volID limit is 128 https://github.com/ceph/ceph-csi/blob/devel/internal/util/volid.go#L53

Reply:
So 64 + 128 ?

Reply:
the clusterID max length is 64 means buildClusterID returned string should be less than 64

Reply:
why not always hash it instead of hashing only if the length is exceeding?

Reply:
I thought we said it would be useful to see a non-hashed value if possible.

Reply:
yeah, I remember but having hash always avoids confusion (the clusterID always remains hash in whatever the case) even if a user creates the filesystem/subvolumegroup/cluster namespace with bigger or smaller names?

Reply:
Changed to use a hash only